benchmarks/sign.mjs

```diff
@@ -9,7 +9,7 @@ async function runSuites() {
     compareSigning({ a: 1, b: 2, c: 3 }, algorightm, privateKeys[algorightm], publicKeys[algorightm])
     return
   } else {
-    for (const algorightm of ['HS512', 'ES512', 'RS512', 'PS512', 'EdDSA']) {
+    for (const algorightm of ['HS256', 'HS512', 'ES512', 'RS512', 'PS512', 'EdDSA']) {
```

Comment:
To avoid doing this change again in the future I would extract the algorithms array in a new file that can be also imported by `verify.mjs`

Reply:
Also, I would fix the typo: `algorithm` instead of `algorightm`